Virtual Google Assistant Typescript issues on missing declarations on the Npm package

### Description:
Reported by @armonge on gitter:
> Hi guys, i've been trying to use virtual-google-assistant with a typescript project, seems like right now there are a couple declarations missing from the npm package

```
 File change detected. Starting incremental compilation... 
node_modules/virtual-google-assistant/lib/src/Index.d.ts:1:71 - error TS7016: Could not find a declaration file for module './VirtualGoogleAssistant'. '/home/armonge/workspace//node_modules/virtual-google-assistant/lib/src/VirtualGoogleAssistant.js' implicitly has an 'any' type. 1 export { VirtualGoogleAssistant, VirtualGoogleAssistantBuilder } from "./VirtualGoogleAssistant"; ~~~~~~~~~~~~~~~~~~~~~~~~~~ node_modules/virtual-google-assistant/lib/src/Index.d.ts:2:31 - error TS7016: Could not find a declaration file for module './ActionInteractor'. '/home/armonge/workspace//node_modules/virtual-google-assistant/lib/src/ActionInteractor.js' implicitly has an 'any' type. 2 export { RequestFilter } from "./ActionInteractor"; ~~~~~~~~~~~~~~~~~~~~ [17:21:08] Found 2 errors. Watching for file changes.
```
> this happens when using
> "strict" : true
> in the tsconfig

### Environment:
- Version: TBD
- OS: TBD
- Node version: TBD

### Steps To Reproduce
These issue needs to be reproduced first, but should just be to import the library on a project on typescript project and then add "strict": true on the tsconfig

### Expected behavior
The library works as expected

### Actual behavior
An exception is thrown due to missing definitions

--- a/package.json
+++ b/package.json
@@ -4,14 +4,13 @@
   "private": false,
   "version": "0.2.6",
   "main": "./lib/src/Index.js",
-  "typings": "./lib/src/Index.d.ts",
+  "types": "./lib/src/Index.d.ts",
   "engines": {
     "node": "> 6.0.0"
   },
   "files": [
     "lib/src/*.js",
-    "lib/src/Index.d.ts",
-    "lib/src/VirtualAlexa.d.ts"
+    "lib/src/*.d.ts"
   ],
   "nyc": {
     "exclude": [

